File: easy/arithmetic-operator/Python/arithmetic_operator.py
def add(a, b) -> int:
    return a + b


def minus(a, b) -> int:
    return a - b


def product(a, b) -> int:
    return a * b


if __name__ == '__main__':
    a = int(input("First number: "))
    b = int(input("Second number: "))

    print(add(a, b))
    print(minus(a, b))
    print(product(a, b))

File: easy/arithmetic-operator/JavaScript/code/arithmetic-operator.ts
export function add(a: number, b: number): number {
	return a + b;
}
export function minus(a: number, b: number): number {
	return a - b;
}

export function product(a: number, b: number): number {
	return a * b;
}

function arithmeticOperator(a: number, b: number) {
	console.log(add(a, b));
	console.log(minus(a, b));
	console.log(product(a, b));
}
arithmeticOperator(3, 5);
